scrapy/downloadermiddlewares/httpcompression.py

```diff
@@ -73,7 +76,10 @@ def _decode(self, body, encoding):
                 # http://www.gzip.org/zlib/zlib_faq.html#faq38
                 body = zlib.decompress(body, -15)
         if encoding == b'br' and b'br' in ACCEPTED_ENCODINGS:
-            body = brotli.decompress(body)
+            if (flag):
+                body = brotli.decompress(body)
+            else:
+                print("brotli not installed")
         if encoding == b'zstd' and b'zstd' in ACCEPTED_ENCODINGS:
```

Comment:
You should apply the same condition as above here, as it falls in the same case/issue (not having the dependency)